settings.gradle

```diff
@@ -5,8 +5,8 @@ import org.gradle.api.GradleScriptException
 def minGradleVersion = GradleVersion.version("4.4")
 // Minimum Gradle version for JUnit5
 def minJunit5GradleVersion = GradleVersion.version("4.6")
-// Minimum Gradle version for builds of JavaFX 11 module
-def minFxGradleVersion = GradleVersion.version("4.10")
+// Minimum Gradle version for builds of JavaFX 17 module using JDK 17
+def minFxGradleVersion = GradleVersion.version("7.3")
```

Comment:
Is this Gradle version bump necessary? Why?

Reply:
Gradle 7.3 is required to support JDK 17.